src/Core/Config/ConfigInterface.php

```diff
@@ -28,4 +28,8 @@ public function getGlobalCss(): array;
     public function getGlobalJs(): array;
 
     public function getAssets(): \Traversable;
+
+    public function getCid(): string;
```

Comment:
I think it'd be nice to have a doc comment explaining what `cid` is; even if the user is never going to provide one, they might not know that they don't need to if they aren't sure what `cid` is